File: 10102.cpp
#include <iostream>
using namespace std;

int main() {
	int V; cin >> V;
	string s; cin >> s;

	int score = 0;
	for (int i = 0; i < V; i++) {
		if (s[i] == 'A') score--;
		else score++;
	}

	if (score == 0) cout << "Tie\n";
	else if (score < 0) cout << "A\n";
	else cout << "B\n";
}

File: 10102.py
V = int(input())
s = input()

score = 0
for i in range(V):
    if s[i] == 'A': score -= 1
    else: score += 1

if score == 0: print("Tie")
elif score < 0: print("A")
else: print("B")
